### PR TITLE
updated title-meta-tags and naming convention

### PR DESCRIPTION
**Pull request by vinay-coder1**

### CODE REVIEW

File: src/main/webapp/WEB-INF/web.xml [hunk 1]
```diff
@@ -4,6 +4,6 @@
          xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/web-app_3_1.xsd"
          version="3.1">
   <welcome-file-list>
-    <welcome-file>index.html</welcome-file>
+    <welcome-file>X-workz-Software-Training-Center.html</welcome-file>
```
Comment (xworkzodc) [inline src/main/webapp/WEB-INF/web.xml]: can we keep it as X-workz-java-testing-development-software-training-center

File: src/main/webapp/X-workz-Software-Training-Center.html [hunk 2]
```diff
@@ -0,0 +1,355 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+<title>X-workz</title>
```
Comment (xworkzodc) [inline src/main/webapp/X-workz-Software-Training-Center.html]: we need to add more-info in title ?

File: src/main/webapp/X-workz-Software-Training-Center.html [hunk 3]
```diff
@@ -0,0 +1,355 @@
...
+java development , Testing ,Web development. We also offer free classes for core java , we train on full-stack development with Enterprise Application development">
+
+<link rel="shortcut icon" href="assets/images/Logo.png" />
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+<meta http-equiv='cache-control' content='no-cache'>
+<meta http-equiv='expires' content='0'>
+<meta http-equiv='pragma' content='no-cache'>
+
+<title>Software Development Training Center-learn java development with Certification</title>
```
Comment (xworkzodc) [inline src/main/webapp/X-workz-Software-Training-Center.html]: isn't this duplicae?? There is title above as well

File: src/main/webapp/X-workz-Software-Training-Center.html [hunk 4]
```diff
@@ -0,0 +1,355 @@
...
+<meta name="keywords"
+	content="Java training institute, Best Sofware development training center in banglore, web development training ,java training institutes near me ,
```
Comment (xworkzodc) [inline src/main/webapp/X-workz-Software-Training-Center.html]: add lot of things , like angularm testing, manual , selenium, sql, web, java frameworks, java projects, Rest web services, interviews, placements , internships..

File: src/main/webapp/X-workz-gallery-placements-photos.html [hunk 5]
```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+<title>X-workz-Best IT placement center in bangalore rajajinagar</title>
```
Comment (xworkzodc) [inline src/main/webapp/X-workz-gallery-placements-photos.html:5]: explore more key words from training perspective you can check with Vinay

File: src/main/webapp/about-X-workz-trainers-courses.html [hunk 6]
```diff
@@ -0,0 +1,256 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+<title>X-workz-Software development Courses-Trainers-software Developer jobs-Team-History of X-workz</title>
+<meta name="description="
+	content="X-workz is software training center ,we train the students on
```
Comment (xworkzodc) [inline src/main/webapp/about-X-workz-trainers-courses.html:7]: like this , can we use in all pages ..???

File: src/main/webapp/X-workz-Software-Training-Center.html [hunk 7]
```diff
@@ -0,0 +1,355 @@
...
+	 java training institutes in bangalore, java training institute in btm layout,java training institute in rajajinagar, free online courses with free certificates, 
+	 free online course java, java tutorials, java programming basics ,angular js tutorials for beginners" />
+
+<meta name="keywords"
+	content="Institute Responsive web template, Bootstrap Web Templates, Flat Web Templates, Android Compatible web template,
```
Comment (xworkzodc) [inline src/main/webapp/X-workz-Software-Training-Center.html]: why these keywords **  Compatible web template ????

File: src/main/webapp/X-workz-Software-Training-Center.html (hunk 4 above)
Comment (xworkzodc) [reply src/main/webapp/X-workz-Software-Training-Center.html]: Please add comments in html on what bases keywords are used, please

File: src/main/webapp/X-workz-gallery-placements-photos.html (hunk 5 above)
Comment (vinay-coder1) [reply src/main/webapp/X-workz-gallery-placements-photos.html:5]: yes sir, I talked to him and added some of our client company names

File: src/main/webapp/about-X-workz-trainers-courses.html (hunk 6 above)
Comment (vinay-coder1) [reply src/main/webapp/about-X-workz-trainers-courses.html:7]: yes sir, I added

File: src/main/webapp/X-workz-Software-Training-Center.html (hunk 7 above)
Comment (vinay-coder1) [reply src/main/webapp/X-workz-Software-Training-Center.html]: its old code which was already there , and I changed it some pages

File: src/main/webapp/X-workz-gallery-placements-photos.html (hunk 5 above)
Comment (ombn) [reply src/main/webapp/X-workz-gallery-placements-photos.html:5]: Which client Name

File: src/main/webapp/X-workz-gallery-placements-photos.html [hunk 8]
```diff
@@ -1,153 +1,153 @@
...
+										This is x-workz development center</marquee>
+								</div>
+							</div>
+							<div class="navbar-collapse collapse bottom-align"
+								id="bs-example-navbar-collapse-1">
+								<ul class="nav navbar-nav navbar-right">
+									<li><a href=" X-workz-java-testing-development-software-training-center.html">Home</a></li>
+									<li><a href="about-X-workz-trainers-courses.html">About</a></li>
+									<li><a href="courses-available-at-X-workzs.html">Courses</a></li>
+									<li><a href="contact-X-workz-address-location.html">Contact</a></li>
```
Comment (ombn) [inline src/main/webapp/X-workz-gallery-placements-photos.html:82]: spelling mistake

File: src/main/webapp/X-workz-java-testing-development-software-training-center.html [hunk 9]
```diff
@@ -1,348 +1,356 @@
...
-
-</html>
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+<!-- Page title and description contents are displayed in search results -->
+<title>Software Development Training Center-learn java development with Certification</title>
+<meta name="description=" content="X-workz is software training center ,we train the students on
+java development , Testing ,Web development. We also offer free classes for core java , we train on full-stack development with Enterprise Application development">
```
Comment (ombn) [inline src/main/webapp/X-workz-java-testing-development-software-training-center.html:8]: mention more technologies angular,sql, web , selenium , projects, internships...college placements etc,,,,

File: src/main/webapp/about-X-workz-trainers-courses.html (hunk 6 above)
Comment (ombn) [reply src/main/webapp/about-X-workz-trainers-courses.html:7]: **I dont see  full meta-info in all pages**

File: src/main/webapp/referral-X-workz-companies-emailId.html [hunk 10]
```diff
@@ -1,273 +1,273 @@
...
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+<title>X-workz-referral interviews</title>
+<link rel="shortcut icon" href="assets/images/Logo.png" />
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+<meta name="keywords"
+	content="The 10 Best Tech Jobs,BE jobs, Software Developer, Data Scientist, Information Security Analyst, Computer Systems Analyst, Web Developer, Sales Engineer,  Information Technology Manager, Computer Research Scientist,
```
Comment (ombn) [inline src/main/webapp/referral-X-workz-companies-emailId.html]: we dont train on these....use only which is preferred

**Review by ombn — CHANGES_REQUESTED** [review]

File: src/main/webapp/X-workz-java-testing-development-software-training-center.html (hunk 9 above)
Comment (vinay-coder1) [reply src/main/webapp/X-workz-java-testing-development-software-training-center.html:8]: used in different pages sir

File: src/main/webapp/referral-X-workz-companies-emailId.html (hunk 10 above)
Comment (vinay-coder1) [reply src/main/webapp/referral-X-workz-companies-emailId.html]: I tried to give different meta-info on each page sir

File: src/main/webapp/X-workz-gallery-placements-photos.html (hunk 8 above)
Comment (vinay-coder1) [reply src/main/webapp/X-workz-gallery-placements-photos.html:82]: keywords used in the contact page

**Pull request closed without merging**